### ISSUE TITLE
health.graphics is missing Nimbledroid uploads

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#12]
### Why/User Benefit/User Problem
https://health.graphics/android does not have the latest Nimbledroid uploads.

### Impact
Since most users will probably go there – rather than Nimbledroid – to see the latest status, they'll be getting out of date numbers.

### Acceptance Criteria (how do I know when I’m done?)
- Nightly builds are regularly uploaded to health.graphics/android

**Comment by mcomella** [comment]
I filed a health.graphics issue upstream: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/591

Assigning to myself to take care of keeping up-to-date with the status of this issue.

**Comment by mcomella** [comment]
Nightly builds have been fixed so this should be unblocked for action from the dashboard team.

**Comment by mcomella** [comment]
They fixed the back-end: https://github.com/mozilla/firefox-health-backend/issues/71#event-2833466981

And are working on updating the front-end: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/591

**Comment by mcomella** [comment]
Note that until we upload Chrome Beta to the admin account, we also will be missing Chrome numbers.

**Comment by mcomella** [comment]
This isn't support important right now so I'm going to unassign and send to triage.

**Comment by mcomella** [comment]
Not important right now: moving to unimportant.

**Comment by mcomella** [comment]
(copy-paste) Not a priority: closing.
